libraries/rush-lib/src/cli/actions/ScanAction.ts

```diff
@@ -56,14 +61,31 @@ export class ScanAction extends BaseConfiglessRushAction {
       parameterLongName: '--all',
       description: 'If this flag is specified, output will list all detected dependencies.'
     });
+    this._folders = this.defineStringListParameter({
+      parameterLongName: '--folder',
+      parameterShortName: '-f',
+      argumentName: 'FOLDER',
+      description:
+        'The folders that need to be scanned, default is src and lib.' +
+        'Normally we can input all the folders under the project directory, excluding the ignored folders.'
+    });
```

Comment:
Stick the default value here.

Reply:
Don't quite understand what this is for.